core/blockchain.go

```diff
@@ -209,7 +209,8 @@ func (self *BlockChain) loadLastState() error {
 		return err
 	}
 	self.stateCache = statedb
-	self.stateCache.GetAccount(common.Address{})
+	// XXX Reviewers: what's this for? I need to know if I need to backport this somehow
```

Comment:
My guess would be that some other code depends (or depended) on that address being in the state cache for some reason.